HelpSource/scdoc.css

```diff
@@ -238,10 +238,10 @@ a.menu-link:hover {
     float: right;
     font-size: 12pt;
     padding: 0.4em 0.4em 0.2em 0.4em;
-    background-color: var(--sc-tertiary);
-    color: white;
+    background-color: var(--sc-bg-warning);
```

Comment:
Sorry for the misunderstanding, I didn't mean to remove the default extensions color, but rather to put back `#601c8b`